### PR TITLE
create known relationship

### PR DESCRIPTION
**Pull request by joshcrews**

### CODE REVIEW

**Review by taylorbrooks — CHANGES_REQUESTED** [review]
@joshcrews I'm getting failed specs as a result of this PR.

**Comment by joshcrews** [comment]
I pushed a spec fix

File: lib/rock_rms/resources/group_member.rb [hunk 1]
```diff
@@ -19,6 +19,16 @@ def delete_group_member(id)
...
       end
 
+      def create_known_relationship(person_id:, related_person_id:, relationship_role_id:)
+        url_params = {
+          personId: person_id,
+          relatedPersonId: related_person_id,
+          relationshipRoleId: relationship_role_id
+        }
+
+        post("GroupMembers/KnownRelationship?#{URI.encode_www_form(url_params)}")
```
Comment (taylorbrooks) [inline lib/rock_rms/resources/group_member.rb:29]: Just curious, do you have to use the `URI.encode_www_form` method call?

I was under the impression that Faraday would just magically figure it out if you passed two arguments to `post()`

```ruby
params = {
  personId: person_id,
  relatedPersonId: related_person_id,
  relationshipRoleId: relationship_rold_id
}
post('GroupMembers/KnownRelationship', params)
```

Comment (joshcrews) [reply lib/rock_rms/resources/group_member.rb:29]: faraday magic does figure it out, but it figures out to use form params and Rock 500 errors on it.

what is needed is POST but with the params in the url

Comment (taylorbrooks) [reply lib/rock_rms/resources/group_member.rb:29]: Gotcha!

Thanks, I learned something today.

**Pull request merged**